time stamp
float64[] vel_target
float64[] vel_now
float64[] pos_target
float64[] pos_now
float64[] current